docs/index.adoc

```diff
@@ -23,7 +23,7 @@ Given a _start_ object, korrel8r searches for a chain of related rules, and find
 Korrel8r can be run as a plain executable, outside the cluster, and connect to the API server and stores inside the cluster. It can also run as a deployed service inside the cluster.
 
 [id="index-architecture"]
```

Comment:
I would appreciate using IDs for headings since this helps with links / xrefs and is closer to downstream formatting, but I can open a PR to add these later.

Reply:
+1 I'll try to do it as I go but feel free to correct.